doc/source/cluster/kubernetes/user-guides/kubectl-plugin.md

```diff
@@ -60,9 +69,250 @@ Forwarding from [::1]:10001 -> 10001
 ### Get Ray cluster logs
 
 ```shell
-$ kubectl ray logs rayjob-sample-raycluster-kfhl6
+$ kubectl ray logs sample-raycluster-kfhl6
 No output directory specified, creating dir under current directory using cluster name.
 Command set to retrieve both head and worker node logs.
-Downloading log for Ray Node rayjob-sample-raycluster-kfhl6-head-87xpb
-Downloading log for Ray Node rayjob-sample-raycluster-kfhl6-small-group-worker-54qfm
+Downloading log for Ray Node sample-raycluster-kfhl6-head-87xpb
+Downloading log for Ray Node sample-raycluster-kfhl6-small-group-worker-54qfm
+```
+
+#### Get Ray job logs
+
+Users can also Ray logs with RayJobs and RayServices
+
+```shell
+$ kubectl ray logs rayjob/rayjob-interactivemode
+No output directory specified, creating dir under current directory using resource name.
+Command set to retrieve both head and worker node logs.
+Downloading log for Ray Node rayjob-interactivemode-raycluster-qbkr8-head-dwm84
+Downloading log for Ray Node rayjob-interactivemode-raycluster-qbkr8-small-grou-worker-hr2jp
+```
+
+### Submit A Ray Job
+
+This is a wrapper around the `ray job submit` command. The plugin can automatically forward the ports to the Ray cluster and submit the job.
```

Comment:
We should add more details here that running `kubectl ray job submit` not only runs `ray job submit`, but also provisions the ephemeral cluster used to execute the job

Reply:
Changed, PTAL!